README.md

```diff
@@ -2,23 +2,29 @@
 
 [![Go Reference](https://pkg.go.dev/badge/github.com/google/osv-scalibr.svg)](https://pkg.go.dev/github.com/google/osv-scalibr)
 
-**Note:** The code in this repo is subject to change in the near future as we're merging SCALIBR with [OSV-scanner](https://github.com/google/osv-scanner) to provide a single tool that unifies the two scanners' extraction and vuln scanning capabilities.
-
-SCALIBR (Software Composition Analysis Library) is an extensible file system scanner used to extract software inventory data (e.g. installed language packages) and detect vulnerabilities.
+OSV-SCALIBR (Software Composition Analysis Library) is an extensible library providing:
+- File system scanner used to extract software inventory data (e.g. installed language packages) and detect known vulnerabilities or generate SBOMs. 
```

Comment:
There's some trailing whitespace at the end of this line that our internal linter is complaining about
(I wonder if there's an external linter for that we could enable)